stubs/olefile/olefile/olefile.pyi

```diff
@@ -172,10 +173,14 @@ class OleDirectoryEntry:
     def getmtime(self) -> datetime.datetime | None: ...
     def getctime(self) -> datetime.datetime | None: ...
 
+_Property: TypeAlias = int | str | bytes | bool | None
+
 class OleFileIO:
```

Comment:
```suggestion
class OleFileIO[Generic[AnyStr]:
```

One problem I see is that the type var can't be determined statically from `__init__`, unless `bytes` or `str` is used. So we will need an overload for the `IO[bytes]` and `None` cases.